### ISSUE TITLE
As a member of the public, I want to learn about Justice40.

### ISSUE THREAD

**Issue by switzersc-usds** [usds/justice40-tool#16]
**Description**
We should put up a lightly-styled, representative-content version of our initial site for the sake of forming an informational landing page. 

**Solution**
Following the creation of our static site (https://github.com/usds/justice40-tool/issues/17), we should add some approved content and some basic styling utilizing the US Web Design System.

**Describe alternatives you've considered**
* Not posting any content to begin with -- we need to give the public a feel for the messaging and design of our eventual site right out of the gate. 

**Links to user research or other resources**
N/A

**General Tasks**
- [ ] Get content approved
- [ ] Get translation approved

**Tech Tasks**
- [x] Re-generate Gatsby base page with minimal plugin setup- @NatHillardUSDS 
- [x] Integrate [gatsby-plugin-intl](https://www.gatsbyjs.com/plugins/gatsby-plugin-intl/?=i18n) for internationalization - @NatHillardUSDS 
- [x] Page Layout - @TomNUSDS 
- [x] Mobile support - @TomNUSDS 
- [x] Code cleanup to remove older map libraries - @NatHillardUSDS 
- [x] Implement accessibility best practices a la [here](https://www.gatsbyjs.com/docs/conceptual/making-your-site-accessible/) - @NatHillardUSDS 
- [x] Add styles via sass and module imports
- [x] Add [@trussworks/react-uswds](https://github.com/trussworks/react-uswds) - @TomNUSDS  
- [x] Configure Typescript - @NatHillardUSDS 

**Definition of "Done"**
- [ ] We've posted content in English and Spanish
- [x] We've styled that content appropriately according to a related spec. 
- [x] We have a website we can spin up locally. [This card is not about deploying]

**Design Needs**
* Reduced version of site spec (perhaps a limited version of earlier-discussed prototype, with a focus on content and content placement?)

**Comment by switzersc-usds** [comment]
Aiming to have wireframes by EOW.

**Comment by janelien** [comment]
Content is still a work in progress

![J40_informational site.png](https://images.zenhubusercontent.com/60998bc7b85eebae77bd2048/9bcef99b-861c-4e02-8f49-da261760295d)

![J40_informational site_mobile.png](https://images.zenhubusercontent.com/60998bc7b85eebae77bd2048/65b0b7c3-4d33-4590-af02-4e9698010abb)

**Comment by switzersc-usds** [comment]
Next steps: @NatHillardUSDS will set up dev+design sync on USWDS limitations for styles. We may be able to expect official Spanish content translation sometime next week.

**Comment by switzersc-usds** [comment]
Footer design is subject to change based on domain decision.

**Comment by katherinedm-usds** [comment]
Related/dependent Spanish translation ticket is https://github.com/usds/environmental-justice/issues/155

**Comment by TomNUSDS** [comment]
This is mostly done layout wise. Still need to fix some figure out some font issues and padding on mobile, but these can probably just become bugs.

Not sure about where it's going to be hosted. Think we're waiting on a domain name? Or, is the plan to just have it on github for review only?

Language selection is missing as is a nav menu. But we also don't have any translations or other pages yet, so those could be tied to these other features.

**Comment by switzersc-usds** [comment]
@janelien will bump on content approval Monday, then this can go out for translation.

**Comment by switzersc-usds** [comment]
We are just blocked by the translation piece, so I've created a new ticket for that and am closing this one. https://github.com/usds/justice40-tool/issues/184
